packages/devtools_app/lib/src/performance/flutter_frames_chart.dart

```diff
@@ -330,11 +338,11 @@ class FlutterFramesChartItem extends StatelessWidget {
     );
   }
 
-  String _tooltipText(FlutterFrame frame) {
+  String _tooltipText(FlutterFrame frame, bool hasShaderJank) {
     return [
       'UI: ${msText(frame.uiEventFlow.time.duration)}',
       'Raster: ${msText(frame.rasterEventFlow.time.duration)}',
-      if (frame.hasShaderTime) 'Skia.Shaders: ${msText(frame.shaderDuration)}',
+      if (hasShaderJank) ...['Skia.Shaders: ${msText(frame.shaderDuration)}'],
```

Comment:
Lets give this a clearer name than Skia.Shaders. Perhaps `Shader Compilation` or use other terminology consistent with the documentation on early onset jank.

Reply:
done